Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/BaseMousePointer.cs

```diff
@@ -59,23 +59,14 @@ public abstract class BaseMousePointer : BaseControllerPointer, IMixedRealityMou
 
         protected abstract string ControllerName { get; }
 
-        private IMixedRealityController controller;
-
         /// <inheritdoc />
         public override IMixedRealityController Controller
         {
-            get { return controller; }
+            get => base.Controller;
             set
             {
-                controller = value;
+                base.Controller = value;
                 TrackingState = TrackingState.NotApplicable;
-
-                if (controller != null && gameObject != null)
```

Comment:
This is removed but seems important. Can you explain why you removed it?

Reply:
It's not removed. It's executed via `base.Controller = value` inside of BaseControllerPointer